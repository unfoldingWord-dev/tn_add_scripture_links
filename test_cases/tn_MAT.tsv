Reference	ID	Tags	SupportReference	Quote	Occurrence	Note
1:1	abc1		rc://*/ta/man/translate/translate-names	Jesus	1	This genealogy connects to Genesis 22:18 and 2 Samuel 7:12-16.
1:2	abc2		rc://*/ta/man/translate/translate-names	Abraham	1	See Genesis 12:1-3 and compare chapter 2.
2:1	abc3		rc://*/ta/man/translate/translate-names	Bethlehem	1	This fulfills Micah 5:2 and verses 4-6.
