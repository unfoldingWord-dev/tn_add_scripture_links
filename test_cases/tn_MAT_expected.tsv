Reference	ID	Tags	SupportReference	Quote	Occurrence	Note
1:1	abc1		rc://*/ta/man/translate/translate-names	Jesus	1	This genealogy connects to [Genesis 22:18](../../gen/22/18.md) and [2 Samuel 7:12-16](../../2sa/07/12.md).
1:2	abc2		rc://*/ta/man/translate/translate-names	Abraham	1	See [Genesis 12:1-3](../../gen/12/01.md) and compare [chapter 2](../02/01.md).
2:1	abc3		rc://*/ta/man/translate/translate-names	Bethlehem	1	This fulfills [Micah 5:2](../../mic/05/02.md) and [verses 4-6](../02/04.md).
